apps/postybirb-ui/src/components/form/fields/field-translations.ts

```diff
@@ -0,0 +1,17 @@
+import { MessageDescriptor } from '@lingui/core';
+import { msg } from '@lingui/macro';
+import { FieldTranslations } from '@postybirb/types';
+
+export const fieldLabelTranslations: {
+  [K in keyof FieldTranslations]: MessageDescriptor;
+} = {
+  description: msg`Description`,
+  tags: msg`Tags`,
+  title: msg`Title`,
+  rating: msg`Rating`,
+  species: msg`Species`,
+  contentWarning: msg`Content warning`,
+  useThumbnail: msg`Use thumbnail`,
+  allowResize: msg`Allow resizing image`,
+  feature: msg`Feature`,
```

Comment:
Im not really sure how to do things like this, maybe it will be worth moving all the external translations into the translation lib.

However i want it to be expandable, so for example sites can add translations they use (fa - species, da - feature, send to sraps etc), and because of that i must keep it use interface FieldTranslations

Also by keeping it in the one place we ensuring that every single entry is present so no "Translation not found" will happen

Maybe i should do the same for the validation fields, im not really sure

Reply:
Yeah, this is kind of why I went with the field.label || field.translatedLabel route to at least always have a fallback, even if to english.

But ideally we will catch any missing translations for new websites and fields.